ubiquity/pay.ubq.fi
# PR: pick & offer the single best virtual card to user

Resolves https://github.com/ubiquity/pay.ubq.fi/issues/273

How to test the best card based on location? 
You can either use a vpn to change your IP to see the result. Or, you can also use adblock like ublock origin to block ip location tracking, and fallback to your OS timezone. Change OS timezone to see results. Just so you know, permit amount also matters. If your location results in a mastercarad international, but your reward permit is too low for it to mint, you get a next option if any in the amount range. 

## QA


https://github.com/user-attachments/assets/905fc692-e4fc-463a-9c52-469c5b6e12e7
